lib/src/clojure_lsp/feature/diagnostics.clj

```diff
@@ -213,6 +214,39 @@
 (defn publish-empty-diagnostics! [uris components]
   (publish-all-diagnostics!* components (map empty-diagnostics-of-uri uris)))
 
+(defn ^:private namespace-alias->finding [element inconsistencies kondo-config]
+  {:uri (:uri element)
+   :row (:name-row element)
+   :col (:name-col element)
+   :end-row (:name-end-row element)
+   :end-col (:name-end-col element)
+   :level (-> kondo-config :linters :clojure-lsp/uniform-aliasing :level)
+   :message (format "Different aliases %s found for %s"
```

Comment:
I wonder if the message can match the linter name or vice versa for better understanding, maybe the linter should be `different-aliases` or `divergent-aliases`?

Reply:
No prob, I'll go with `different-aliases`.